### ISSUE TITLE
Stuck: Pharao summoned mummy outside the map

### ISSUE THREAD

**Issue by askebv** [Heishe/idleraiders-second-run#532]
![image](https://cloud.githubusercontent.com/assets/4981659/23876669/e36f0c8a-083e-11e7-9bf7-fca93fffc4bf.png)

This left my raiders stuck trying to get to it, and it couldn't get inside either.

This happened because I reset the raid after losing to the boss (while still viewing the raids, I pressed rest and then started this particular raid right away). And then I think ignite was still attached to it after the reset, so my tank threw a saving throw and it started walking towards the raiders, and we ended up fighting in the narrow hallway instead.

**Comment by Meissner-Effect** [comment]
Yes, this happened to me as well even though I didn't reset the map. Pharao kept gradually moving further into the map until he summoned a mummy outside the map in the same spot.
